themes/palenight-italic.json

```
@@ -1264,6 +1308,14 @@
         "foreground": "#82b1ff"
       }
     },
+    {
+      "name": "Asciidoc Function",
+      "scope": "entity.name.function.asciidoc",
+      "settings": {
+        "foreground": "#F78C6C",
+        "fontStyle": ""
+      }
```

(same across the other variants)

```suggestion
      "settings": {
        "foreground": "#F78C6C"
      }
```

---

It means the attributes will keep the bold or italic formatting from the bloc they are describing, I'm OK with your suggestions, I will includes them.

---

I applied this to all the variants like suggested :-)